concepts/Vulnerability_disclosure_and_embargo_policy/00004-Vulnerability_disclosure_and_embargo_policy.md

```diff
@@ -0,0 +1,221 @@
+- Tags: security, infrastructure
+
+## Summary
+
+Describe the security vulnerability disclosure process, including embargoes.
+
+## Motivation
+
+Users and researchers who discover security issues in Enarx need to know that
+we welcome their efforts and how we will manage their disclosure. They must
+also be able to have an idea of how we use security embargoes.
+
+In other words:
+- We want:
+  - to encourage reporting of security issues (ie: “attack us!”)
+  - to make it easy to report security issues
+  - to maintain and extend our policy of safe defaults / discouraging people
+  from adopting bad security practices
+  - to recognise and show appreciation for the time spent researching and
+  reporting
+  - to make it possible for people who may be at risk to report issues
+  pseudonymously
+
+- We need:
+  - a clear policy, describing both:
+    - how to report; and
+    - what to expect during the process
+  - a main reporting channel:
+    - secondary channels, should the main one fail/not be available/be
+  overloaded.
+  - a list of people who can be reached directly
+  - a place to list security reports and give attribution
+
+
+## Tutorial
+
+This RFC aims to provide a set of documents (wiki pages) that describe the 
+Enarx vulnerability disclosure policy and the list of security advisories and
+ fixes.
+
+The two pages are "Vulnerability Reporting and Embargo Policy" and "Enarx 
+Security Advisories"
+
+--- 
+### Vulnerability Reporting and Embargo Policy
+The Enarx project aims to be as safe to use as possible and therefore to 
+produce code that is as secure as possible. As such, we welcome any and all 
+research into Enarx and appreciate your efforts to take the time to 
+responsibly disclose any issues you find.
+
+Vulnerabilities may be discovered in:
+1. Enarx code
+2. Enarx dependencies (including compilers)
+3. Enarx documentation
+4. 3rd party hardware or firmware
+
+All security bugs in Enarx should be reported by email to security@enarx.io. 
+This list is delivered to a small security team. Your email will be 
+acknowledged within 48 hours, and you’ll receive a more detailed response to 
+your email within 96 hours indicating the next steps in handling your report.
+ If you wish, you may encrypt your report using our public key. This key is 
+also on keys.openpgp.org and reproduced below.
+
+Please ensure that you use a descriptive subject line to avoid having your report be 
+missed (as with any publicly listed addresses, it is expected this one will 
+receive a lot of spam). After the initial reply to your report, the security team
+ will endeavor to keep you informed of the progress being made towards a fix 
+and full announcement. As recommended by 
+[RFPolicy](https://dl.packetstormsecurity.net/papers/general/rfpolicy-2.0.txt),
+ these updates will be sent at least every five working days.
+
+If you have not received a reply to your email within 72 hours, or have not 
```

Comment:
Is this 72 supposed to be 48 or 96, as above?

Reply:
Indeed, thanks. It should be 48 hours, i think.